client/src/common/types/entities.ts

```diff
@@ -1,3 +1,73 @@
 // If a globally shared type or interface doesn't have a clear owner, put it here
 
 export {};
+
+export interface AnnotationColumn {
+  categories?: Category[];
+  name: string;
+  type: "string" | "float32" | "int32" | "categorical" | "boolean";
+  writable: boolean;
+}
+
+interface DataFrame {
```

Comment:
I thought @tihuan was going to fix this name -- it is super confusing as it has nothing to do with a Dataframe.  It is the schema for the X matrix (ie, count matrix).

Reply:
I'm not sure how this file is getting committed by me, I'll take a look

Reply:
bad merge.  He moved this stuff to schema.ts IIRC